src/App.js

```diff
@@ -33,32 +33,69 @@ const NoLogin = () => {
   )
 }
 
+const Error = (error) => {
```

Comment:
I'd prefer a different component name.  Maybe simply "ErrorBoundaryReport" 

This should be `const Error = ({ error }) => {` to conform to the normal structure of a function component.

You'll also need to add 
````js
Error.propTypes = {
  error: PropTypes.object,
}
````
below the component function to keep eslint happy.